CONTRIBUTING.md

```diff
@@ -19,6 +19,8 @@ Pull requests are welcome! If you have questions, feel free to [contact the auth
 Before merging:
 
 - Describe the change in `CHANGELOG.md`
+- Create supporting unit tests.
+- Verify new and existing unit tests pass with `python setup.py nosetests`.
```

Comment:
Should this be `python setup.py test` ?

Reply:
Unfortunately, even with the test runner change I made in setup.py, I couldn't get that command to use nose.

Reply:
`setup.py` test runs unittest

Reply:
So when I do `python setup.py nosetest` I get `error: invalid command 'nosetest'`...

Reply:
:/
Did you do `pip install -r requirements.txt`?

Reply:
Whoops, its `nosetests`, not  `nosetest`. I think I tried the former earlier but that was before I made a new virtualenv for it.